src/generators/validators.cr

```diff
@@ -0,0 +1,13 @@
+module LuckyCli::Generators::Validators
+  module ProjectName
```

Comment:
Could you put this on the line above? It's a small thing but it'd be more in line with the other Lucky classes/modules